cmd/keytransparency-client/cmd/hammer.go

```diff
@@ -41,9 +40,8 @@ func init() {
 
 	RootCmd.AddCommand(hammerCmd)
 
-	hammerCmd.Flags().IntVar(&maxWorkers, "workers", 1, "Number of parallel workers")
+	hammerCmd.Flags().IntVar(&maxWorkers, "workers", 2000, "Number of parallel workers")
```

Comment:
2000 looks a bit excessive a default to me (esp. as a number of workers). Have you considered 100 or 200?

Reply:
I lowered it to 1000. 

Now that the speed of the test is being controlled by `qps`, more workers don't hurt, and only ensure that there is enough parallelism to meet qps needs.